modules/registry-configuring-storage-gcp-user-infra.adoc

```diff
@@ -18,9 +18,33 @@ cloud credentials.
 contents of a credentials file provided by GCP:
 ** `REGISTRY_STORAGE_GCS_KEYFILE`
 
-
 .Procedure
 
-////
-Stub for procedure to manually set storage medium.
-////
+Use the following procedure if the Registry Operator cannot create a GCP bucket
+and automatically configure storage.
+
+. Set up a link:https://cloud.google.com/storage/docs/lifecycle[Lifecycle Management policy]
+to abort incomplete multipart uploads that are one day old.
```

Comment:
incomplete, multipart uploads ? As each adjective applies directly to "uploads".